README.rst

```diff
@@ -238,6 +238,29 @@ The parser use a ``TAG_KEY_MAPPING``, which one can override by calling ``rispy.
     'url',
     'volume']
 
+Other functionality
+-------------------
+Other various utilities included in ``rispy`` are documented below.
+
+Reference type conversion
+*************************
+
+A method is available to convert common RIS reference types into more readable terms. The key for this conversion is located in ``config.py``. An example is located below.
+
+.. code:: python
+
+   >>> from rispy.utils import pretty_reference_types
+   >>> refs = [{"type_of_reference": "JOUR"}]
+   >>> print(pretty_reference_types(refs))
+   [{'type_of_reference': 'Journal'}]
+
+This method has the following parameters:
```

Comment:
could you move these parameters to the function docstring instead of the readme?